### PR TITLE
3.10 Release Notes Updates

### PR DESCRIPTION
**Pull request by bmcelvee**

### CODE REVIEW

**Comment by bmcelvee** [comment]
@gabemontero PTAL

**Review by gabemontero — APPROVED** [review]
LGTM @bmcelvee

**Comment by bmcelvee** [comment]
@ahardin-rh PTAL

**Comment by ahardin-rh** [comment]
LGTM! :shipit:

File: release_notes/ocp_3_10_release_notes.adoc [hunk 1]
```diff
@@ -661,6 +661,13 @@ Jenkins is now updated to 2.107.3-1.1 and Jenkins build agent (slave) images are
 * Node.js 8
 * Maven 3.5
 
+The jenkins-slave-maven-* and jenkins-slave-nodejs-* images are being
```
[the remark this answers is not in the document]
Comment (bmcelvee) [reply release_notes/ocp_3_10_release_notes.adoc]: Right, I just caught that as well. Thanks!

**Pull request merged**